tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -399,7 +450,47 @@ void test_various_containers() {
     test_one_container<list<int>>(); // bidi, not vectorizable
 }
 
+#if _HAS_CXX20
+constexpr bool test_constexpr() {
+    int data[] = {2, 1, 3, 3, 3, 3, 4, 6, 5};

```

Comment:
Nitpick: This shadows `std::data`. I've probably lost this war against shadowing (so many tests are saying `expected`, etc.) but I'll mention it anyways.